### PR TITLE
bump attrs, add simwriter

### PR DESCRIPTION
**Pull request by genric**
@wizmer pls.

### CODE REVIEW

File: patches/additionalPythonPackages/default.nix [hunk 1]
```diff
@@ -1121,4 +1121,80 @@ EOF
     };
   };
 
+  rpy2 = pythonPackages.buildPythonPackage rec {
```
Comment (wizmer) [inline patches/additionalPythonPackages/default.nix]: Ah ! So, patching rpy2 solved yesterday's problem ? Did you copy pasted the one from NixOS ?

File: bbp/nse/simwriter/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    name = "simwriter-${version}";
+
+	src = fetchgitPrivate {
+        url = config.bbp_git_ssh + "/project/proj1/simwriter";
+        inherit rev sha256;
+    };
+
+    propagatedBuildInputs = with pythonPackages; [ matplotlib numpy scipy neurotools cheetah bluepy ];
+
+    doCheck = false;
```
Comment (wizmer) [inline bbp/nse/simwriter/default.nix]: Can you remove this ?

File: patches/additionalPythonPackages/default.nix (hunk 1 above)
Comment (genric) [reply patches/additionalPythonPackages/default.nix]: Kind of copy/paste with some fixes to resolve some names.

File: bbp/nse/simwriter/default.nix (hunk 2 above)
Comment (genric) [reply bbp/nse/simwriter/default.nix]: Done.

**Comment by genric** [comment]
Anything missing here?

**Comment by wizmer** [comment]
Sorry for the delay

**Pull request merged**